llvm/include/llvm/IR/IntrinsicsSPIRV.td

```diff
@@ -14,6 +14,7 @@ let TargetPrefix = "spv" in {
   def int_spv_assign_type : Intrinsic<[], [llvm_any_ty, llvm_metadata_ty]>;
   def int_spv_assign_ptr_type : Intrinsic<[], [llvm_any_ty, llvm_metadata_ty, llvm_i32_ty], [ImmArg<ArgIndex<2>>]>;
   def int_spv_assign_name : Intrinsic<[], [llvm_any_ty, llvm_vararg_ty]>;
+  def int_spv_assign_decoration : Intrinsic<[], [llvm_any_ty, llvm_metadata_ty]>;
```

Comment:
Please update the [SPIRVUsage](https://llvm.org/docs/SPIRVUsage.html#target-intrinsics) document with relevant information about the new intrinsic.

Reply:
Done, thank you

Reply:
Thank you!